File: app/src/main/java/com/example/sony/newsapi/repository/MigrationDB.java
package com.example.sony.newsapi.repository;

import io.realm.DynamicRealm;
import io.realm.RealmMigration;
import io.realm.RealmSchema;

public class MigrationDB implements RealmMigration {
    @Override
    public void migrate(DynamicRealm realm, long oldVersion, long newVersion) {
        RealmSchema schema = realm.getSchema();

        if (oldVersion == 0) {
            schema.create("TaskRealmModel")
                    .addField("title", String.class);
            oldVersion++;
        }
    }
}

File: app/src/main/java/com/example/sony/newsapi/repository/MigrationDB.kt
package com.example.sony.newsapi.repository

import io.realm.DynamicRealm
import io.realm.RealmMigration

class MigrationDB : RealmMigration {
    override fun migrate(realm: DynamicRealm, oldVersion: Long, newVersion: Long) {
        var oldVersion = oldVersion
        val schema = realm.schema

        if (oldVersion == 0L) {
            schema.create("TaskRealmModel")
                .addField("title", String::class.java)
            oldVersion++
        }
    }
}
